realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -203,7 +203,7 @@ public boolean isValid() {
     public RealmQuery<E> isNull(String fieldName) {
         realm.checkIfValid();
 
-        long columnIndices[] = schema.getColumnIndices(fieldName);
+        long columnIndices[] = schema.parseFieldName(fieldName);
```

Comment:
I thought it was better, because to me it seems that it is parsing a (simple) string.  I'm totally happy to go back to the old name, if that seems clearer.  This was just a suggestion.